.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,25 @@
 <!---
-Thank you for contributing to NVIDIA DALI.
+Thank you for contributing to NVIDIA DALI! If you haven't yet,
+please read the contributing guidelines in the CONTRIBUTING.md file.
 
-Please fill the relevant information in this PR template.
+We need a few more information from you to proceed.
+Please fill the relevant sections in this PR template.
 
-Fields with `- [ ]` represent check-boxes that can be marked after you create and save the Pull Request.
+Fields in the Checklist section can be marked after you create and save the Pull Request.
 --->
 
-## Description
-- [ ] **Bug fix** (*non-breaking change which fixes an issue*)
-- [ ] **New feature** (*non-breaking change which adds functionality*)
-- [ ] **Breaking change** (*fix or feature that would cause existing functionality to not work as expected*)
-- [ ] **Refactoring** (*Redesign of existing code that doesn't affect functionality*)
-- [ ] **Other** (*e.g. Documentation, Tests, Configuration*)
 
-### What happened in this PR
+## This PR is a:
+<!--- Please pick one from below. Just uncomment what fits most. --->
+<!--- **Bug fix** (*non-breaking change which fixes an issue*) --->
```

Comment:
Please no. 
First, it does not say how to uncomment.
Second, to uncomment I need to remove manually both ends of the comment marker. It seems like it will be cumbersome on the day to day basis.

Reply:
+1 on uncommenting being cumbersome - copy-paste would be easier.

Reply:
I would assume, that if you are contributing to DALI, you do know how to uncomment this piece of code ;)

And for the latter - I believe that comparing to the effort we should put in the PR descriptions (looking into one of @hugo213 's PRs: #3604... these are really impressing), removing 6 characters is really nothing.

Reply:
Maybe instead of saying "uncomment" just leave the "Please pick one from below" and then one can copy-paste or uncomment depending on what's easier for you